src/App.jsx

```diff
@@ -1,104 +1,30 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import commentsFromServer from './api/comments.json';
+import postsFromServer from './api/posts.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
```

Comment:
The `PostList` component is correctly imported. Ensure that it is implemented to handle the `posts` array with `user` and `comments` properties.